Add example: migrate travis CI to GitHub Action (#3)
sourcegraph/campaign-examples: migrate-travis-to-github-action/README.md

# Migrate from Travis CI to GitHub Actions

> > NOTE: Sourcegraph Campaigns require the [Sourcegraph CLI](https://github.com/sourcegraph/src-cli) be [installed](https://github.com/sourcegraph/src-cli#installation) and [configured](https://github.com/sourcegraph/src-cli#setup) to point at your [Campaigns enabled](https://docs.sourcegraph.com/user/automation#configuration) Sourcegraph instance.

Reviewer: ```suggestion
> NOTE: Sourcegraph Campaigns require the [Sourcegraph CLI](https://github.com/sourcegraph/src-cli) be [installed](https://github.com/sourcegraph/src-cli#installation) and [configured](https://github.com/sourcegraph/src-cli#setup) to point at your [Campaigns enabled](https://docs.sourcegraph.com/user/campaigns) Sourcegraph instance.
```

> This examples migrates from Travis CI to GitHub Actions for Go projects.

Reviewer: ```suggestion
This campaign migrates Go projects from Travis CI to GitHub Actions.
```

> What this does:

Reviewer: ```suggestion
What this action does:
```

1. Delete `.travis.yml` file if exists.

> 2. Copy over the template `go.yml` to `.github/workflows/go.yml` (contains jobs for linting and testing).

Reviewer: ```suggestion
2. Copy over the template `go.yml` from your local machine (on which `src` is being executed) to `.github/workflows/go.yml` (contains jobs for linting and testing) in each repository.
```

---

```json
{

>   "scopeQuery": "file:.*_test.go repo:^github\\.com/gogs/ count:1000",

Reviewer: ```suggestion
  "scopeQuery": "repohasfile:.travis.yml lang:go",
```

Author: Nice!

  "steps": [
    {
      "type": "command",
      "args": ["rm", "-f", ".travis.yml"]
    },
    {
      "type": "command",
      "args": ["mkdir", "-p", ".github/workflows"]
    },
    {
      "type": "command",

>       "args": ["cp", "<path to your directory>/go.yml", ".github/workflows/go.yml"]

Reviewer: ```suggestion
      "args": ["cp", "<path-to-file-on-your-local-machine>/go.yml", ".github/workflows/go.yml"]
```
